requirements/default.txt

```diff
@@ -1,5 +1,5 @@
 # This file was autogenerated by uv via the following command:
-#    uv pip compile --generate-hashes --no-strip-extras requirements/default.in -o requirements/default.txt
+#    uv pip compile --generate-hashes --no-strip-extras default.in -o default.txt
```

Comment:
While CI doesn't fail anymore, please recompile from outside the requirements folder.